### PR TITLE
Adding Gravity bridge token

### PR DESCRIPTION
**Pull request by JeremyParish69**
-assuming the token symbol (GRAV) to be used for 'coinDenom', even though coin denom is 'graviton'.

### CODE REVIEW

**Comment by sunnya97** [comment]
@JeremyParish69 is this good to go?  Or is it still a draft?

File: src/config.ts [hunk 1]
```diff
@@ -2111,4 +2117,38 @@ export const EmbedChainInfos: ChainInfoWithExplorer[] = [
...
 	},
+	{
+		rpc: 'https://gravitychain.io:26657',
+		rest: 'https://gravitychain.io:1317',
+		chainId: 'gravity-bridge-3',
+		chainName: 'Gravity Bridge',
+		stakeCurrency: {
+			coinDenom: 'GRAV',
+			coinMinimalDenom: 'ugraviton',
+			coinDecimals: 6,
```
Comment (sunnya97) [inline src/config.ts:2128]:
```suggestion
			coinDecimals: 6,
			coinGeckoId: 'pool:ugraviton',
```

File: src/config.ts [hunk 2]
```diff
@@ -2111,4 +2117,38 @@ export const EmbedChainInfos: ChainInfoWithExplorer[] = [
...
+		},
+		bip44: {
+			coinType: 118,
+		},
+		bech32Config: Bech32Address.defaultBech32Config('gravity'),
+		currencies: [
+			{
+				coinDenom: 'GRAV',
+				coinMinimalDenom: 'ugraviton',
+				coinDecimals: 6,
```
Comment (sunnya97) [inline src/config.ts:2139]:
```suggestion
				coinDecimals: 6,
				coinGeckoId: 'pool:ugraviton',
```

File: src/config.ts [hunk 3]
```diff
@@ -2111,4 +2117,38 @@ export const EmbedChainInfos: ChainInfoWithExplorer[] = [
...
+				coinImageUrl: window.location.origin + '/public/assets/tokens/grav.svg',
+			},
+		],
+		feeCurrencies: [
+			{
+				coinDenom: 'GRAV',
+				coinMinimalDenom: 'ugraviton',
+				coinDecimals: 6,
```
Comment (sunnya97) [inline src/config.ts:2147]:
```suggestion
				coinDecimals: 6,
				coinGeckoId: 'pool:ugraviton',
```

**Comment by JeremyParish69** [comment]
> @JeremyParish69 is this good to go? Or is it still a draft?

Good to go. Was just waiting for Assetlists PR 99 to be merged first.

**Pull request merged**